### PR TITLE
fix(Member): Parse Member#premiumSince to timestamp

### LINKED ISSUE #1316: Member.premiumsince not converted to epoch timestamp
Both the JSdocs and typescript type definition declare that the `Member#premiumsince` property is a numbers, but [discord gives an ISO timestamp](https://discord.com/developers/docs/resources/guild#guild-member-object-guild-member-structure) and the transforms that the Member class does doesnt match the docs.

Member should show the timestamp as a number

### PR DESCRIPTION
**Pull request by bsian03**
Fixes #1316

### CODE REVIEW

**Comment by DonovanDMC** [comment]
`premium_since` is defined as both optional & nullable in the docs, optional is handled, nullable is not -- this results in NaN

**Comment by bsian03** [comment]
What if you don't want a number smh

**Comment by JustCat80** [comment]
in that case perfect because it is not, luck with your project 👍

**Comment by danny-may** [comment]
From what im reading in the docs, cant the value be `null`, `undefined` or a string? Shouldnt the type guard be a nullish check or a `typeof x === 'string'` check? Strict null equality would break I believe

**Comment by bsian03** [comment]
But.. we already have an undefined check

File: lib/structures/Member.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ class Member extends Base {
             this.activities = data.activities;
         }
         if(data.premium_since !== undefined) {
-            this.premiumSince = data.premium_since;
+            this.premiumSince = data.premium_since === null ? null : Date.parse(data.premium_since);
```
Comment (danny-may) [inline lib/structures/Member.js:84]:
```suggestion
            this.premiumSince = data.premium_since == null ? null : Date.parse(data.premium_since);
```

**Comment by danny-may** [comment]
Ah, on the lines above, I missed that. Ignore me :)

**Review by DonovanDMC — DISMISSED** [review]

**Review by DonovanDMC — CHANGES_REQUESTED** [review]
typings (nullable & optional on both Member & OldMember)
https://github.com/abalabahaha/eris/blob/b2a90ea390bf3f74cec6772cc29956d31d8a3391/index.d.ts#L654
https://github.com/abalabahaha/eris/blob/b2a90ea390bf3f74cec6772cc29956d31d8a3391/index.d.ts#L3046

**Review by DonovanDMC — APPROVED** [review]

**Pull request merged**